File: output/LLaMa/avatar/Python/Java/codeforces_92_A.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        int n = Integer.parseInt(input[0]);
        int m = Integer.parseInt(input[1]);
        int i = 1;
        while (m >= i) {
            m -= i;
            i = (i % n) + 1;
        }
        System.out.println(m);
    }
}

File: output/Qwen_repaired_1/avatar/Java/Python/codeforces_92_A.py
n = int(input().split()[0])
k = int(input().split()[1])
i = 1
req = 1

while k - req >= 0:
    k -= req
    i += 1
    if i % n != 0:
        req = i % n
    else:
        req = n

print(k)